lib/rbac/filterer.rb

```diff
@@ -573,16 +573,18 @@ def lookup_user_group(user, userid, miq_group, miq_group_id)
       miq_group_id ||= miq_group.try!(:id)
       return [user, user.current_group] if user && user.current_group_id.to_s == miq_group_id.to_s
 
+      found_group = user.try(:current_group)
```

Comment:
Is this line needed?

Reply:
Maybe it belongs in the missing `else` before line 583 instead?  Then it can be `found_group = if...` and drop all of the variable setting below.

Reply:
@bdunne 👍  it looks much cleaner